.azure/pipelines/quarantined-pr.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     jobName: MacOS_Quarantined_Test
     jobDisplayName: "Tests: macOS 10.14"
     agentOs: macOS
-    timeoutInMinutes: 60
+    timeoutInMinutes: 120
```

Comment:
/fyi @JunTaoLuo @TanayParikh

Reply:
That job hasn't been timing out has it i.e. weren't the timeouts specific to the quarantined pipeline❔

Reply:
And, oops, forgot you were doing this part

Reply:
Not sure, I thought you meant it was both - I can close https://github.com/dotnet/aspnetcore/pull/25546 if it's just this one

Reply:
@JunTaoLuo @sebastienros @TanayParikh have we seen macOS timeouts in aspnetcore-ci as well as aspnetcore-quarantined-pr❔ I'll dupe this in ci.yml if yes.

Reply:
I haven't paid too much attention to it after my ops rotation so my sample size is limited, but I've seen slowness in macos builds both on aspnetcore-ci and aspnetcore-quarantined-pr.

Reply:
:ok: then I'll change ci.yml too.

Reply:
Hmm, ci.yml doesn't specify timeouts (except for the Helix job) which means its macOS test job gets the default-build.yml default of 180 minutes. Since that's so close to when https://dev.azure.com/dnceng/public/_build/results?buildId=795980&view=results passed, I'll use 240 minutes there.

Reply:
Let me clarify, the aspnetcore-ci run completed after 90 minutes so it still passed without timing out. I think a default timeout of 180 minutes is still fine. I just wanted to mention that build because 90 minutes is more than 3x the usual time of about 25 mins. It's slow, but I don't think we need to adjust our timeout for it.

Reply:
I pushed a change to err on the safe side. The extra half hour timeout is less likely to cause problems elsewhere than having to retry a 1.5 hour job.

Reply:
(Note I didn't double the timeout there😃)

Reply:
> I haven't paid too much attention to it after my ops rotation so my sample size is limited, but I've seen slowness in macos builds both on aspnetcore-ci and aspnetcore-quarantined-pr.

About the same for me as well. Seen occasional slowness for macos specifically.